release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -238,6 +243,15 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-9-removed-features"]
 === Removed features
 
+[id="ocp-4-9-builds-lasttriggeredimageid-parameter"]
+==== The lastTriggeredImageID parameter in builds is removed
+
+This release removes the `lastTriggeredImageID` in the `ImageChangeTrigger` object, which is one of the `BuildTriggerPolicy` types that can be set on a `BuildConfig` spec.
+
+Image change triggers do not start a build based on changes to the `lastTriggeredImageID` field in the `BuildConfig` spec. Instead, the image IDs that trigger a build are recorded in the status of the `BuildConfig` object, which most users cannot change.
```

Comment:
Even cluster admins shouldn't change status fields. That is an easy way to break things.
```suggestion
Image change triggers do not start a build based on changes to the `lastTriggeredImageID` field in the `BuildConfig` spec. Instead, the image IDs that trigger a build are recorded in the status of the `BuildConfig` object.
```